docs/index.md

```
@@ -51,13 +51,30 @@ Debug.Assert(Guid.TryParse(guid, out _));
   <a class="card-grid-item" href="https://testcontainers-python.readthedocs.io/en/latest/">
     <img src="language-logos/python.svg" />Python
   </a>
-  <a class="card-grid-item" href="https://docs.rs/testcontainers/latest/testcontainers/">
+  <a class="card-grid-item" href="https://rust.testcontainers.org/">
     <img src="language-logos/rust.svg" />Rust
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)